src/riak_kv_qry_buffers_ldb.erl

```diff
@@ -84,35 +84,35 @@ add_rows(LdbRef, Rows) ->
     end.
 
 
--spec fetch_rows(eleveldb:db_ref(), non_neg_integer(), unlimited|pos_integer()) ->
+-spec fetch_rows(eleveldb:db_ref(), [{Offset::non_neg_integer(),
+                                      Limit::unlimited|pos_integer()}]) ->
                         {ok, [riak_kv_qry_buffers:data_row()]} | {error, term()}.
-fetch_rows(LdbRef, Offset, LimitOrUnlim) ->
-    %% Because the database is read-only, the plan is to keep a cache
-    %% of iterators for faster seeking (to the nearest stored),
-    %% ideally also enable eleveldb to do the folding from stored
-    %% iterators rather than from 'first'.
+fetch_rows(LdbRef, SortedSpecs) ->
```

Comment:
It looks like you removed the comment explaining the feature that was planned, which makes sense if you implemented it, but by not leaving a comment explaining the feature as it exists now, you've left us with a painful anonymous function to parse for comprehension.